data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1025 @@
+			`	Teeneep concludes the discussion. "I will travel with Captain <last> and see this matter through. I will expect your fleets and armies to be ready to assist us when the time is right." She turns to leave.`
+			choice
+				`	(Return to your ship.)`
+			`	On the way out, Teeneep tells you her next move. "The humans are so eager to raid the facilities that they haven't considered the need for information. We will have to arrange for someone to be 'captured' and brought to Philosopher's Paradise. The human authorities must delay their strike until then.`
+			`	"I'll need an escort back to <destination>. We'll need help from one more group, but I'll contact them on the way. You'll be paid, of course." She hands you <payment> for her most recent journey.`
+				accept
+	on complete
+		payment 400000
+		log `You need to visit the Sunracer spaceport to pick up a man named Tutitot, a worm, and an orb. Tutitot and the worm will act as spies, getting caught by the kidnappers. The orb, placed at Devil-Run, will detect his presence. A week or two after you deposit the orb, the Drak will contact you to let you know Tutitot is ready to report.`
+		conversation
+			`You arrive at an office building with unusually high ceilings. When you enter, you notice one of the Quarg scientists you spoke to before is present, their height explaining the chosen meeting location. This must be the "other group" Teeneep said would help. You and Teeneep join the Quarg and Stephen Moray at a tall table.`
+			`	Teeneep explains her "intentional kidnapping" plan to the others, and Stephen says, "Believe it or not, I met a Hai recently that might be able to help. His name is Tutitot Eeegin, and he's expressed interest in helping us solve this problem. He's clever, and he has some experience acting." Stephen excuses himself to contact Tutitot while the Quarg turns to you and Teeneep.`
+			`	"Teeneep asked if the Quarg have something you could use that would let Tutitot send telepathic signals millions of your kilometers. Alas, the Quarg have been unable to create such technology, nor have the Korath." He smiles. "But the Quarg have been here for eons. In our time, we have discovered... 'devices,' beyond our ability to scientifically explain. One of these would be of use to you." He pulls out a glass orb, and it floats to the table; there seems to be a black worm with purple hairs inside. The orb has a blinking blue light on top.`
+			`	"This worm will let Tutitot talk to the holder of this orb regardless of distance. Bring Tutitot to the kidnappers. Go then to Devil-Eye to place the orb. When Tutitot has reached the Philosopher's Paradise, it will send you a... feeling. You will know it's time."`
```

Comment:
Ah this and the next two are all legacies of the recent rework of it to accommodate for certain other plans having changed in the two years since it was first PR'd. Good catches.

Reply:
Makes me feel a lot better about the confusion I was having, then :)